### DISCUSSION TITLE
How to configure routes as keepalive

### DISCUSSION THREAD

**Discussion by yang4515** [solidjs/solid-start#620] [Q&A]
Hello，I want to configure routes as keepalive. I found a library called solid-keep-alive, but this library can only be used with solid router. Is there any way to configure routes as keepalive in solid-start?

**Comment by ryansolid** [comment]
SolidStart uses `@solidjs/router` but I'm not familiar with the package. Going to move this to questions in discussions.

**Comment by mztlive** [comment]
I also need this ability. Is there any new progress?
This is very useful for admin projects
